lib/src/models/seed_changes/seed_list_diff_changes.dart

```diff
@@ -0,0 +1,182 @@
+
+    return SeedListDiffChange(
+      deletedSeeds: isDeleted ? [seedDiff] : [],
+      addedSeeds: isDeleted ? [] : [seedDiff],
+      deletedKeys: isDeleted ? keys : [],
+      addedKeys: isDeleted ? [] : keys,
+      deletedAccounts: isDeleted ? accounts : [],
+      addedAccounts: isDeleted ? [] : accounts,
+    );
+  }
+
+  /// Build diff changes from seed key (if whole key was added or deleted)
+  factory SeedListDiffChange.fromKey({
+    required SeedKey key,
+    required bool isDeleted,
+  }) {
+    final keyDiff = SeedKeyDiffed.fromKey(key);
+    final accounts =
+        key.accountList.allAccounts.map(KeyAccountDiffed.fromAccount).toList();
+
+    return SeedListDiffChange(
+      deletedSeeds: const [],
+      addedSeeds: const [],
+      deletedKeys: isDeleted ? [keyDiff] : [],
+      addedKeys: isDeleted ? [] : [keyDiff],
+      deletedAccounts: isDeleted ? accounts : [],
+      addedAccounts: isDeleted ? [] : accounts,
+    );
+  }
+
+  /// Build diff changes from key account (if only account was added or deleted)
+  factory SeedListDiffChange.fromAccount({
+    required KeyAccount account,
+    required bool isDeleted,
+  }) {
+    final accountDiff = KeyAccountDiffed.fromAccount(account);
+
+    return SeedListDiffChange(
+      deletedSeeds: const [],
+      addedSeeds: const [],
+      deletedKeys: const [],
+      addedKeys: const [],
+      deletedAccounts: isDeleted ? [accountDiff] : [],
+      addedAccounts: isDeleted ? [] : [accountDiff],
+    );
+  }
+
+  /// Empty state, that could be used as initial state or start value during
+  static const SeedListDiffChange empty = SeedListDiffChange(
+    deletedSeeds: [],
+    addedSeeds: [],
+    deletedKeys: [],
+    addedKeys: [],
+    deletedAccounts: [],
+    addedAccounts: [],
+  );
+
+  /// Added and deleted seeds.
+  /// [SeedDiffed] is just [Seed] where publicKey=masterKey and nothing more.
+  /// Seed have have difference only in 1 way:
+  /// 1) added/removed itself
+  final List<SeedDiffed> deletedSeeds;
+  final List<SeedDiffed> addedSeeds;
+
+  /// Added and deleted public keys.
+  /// [SeedKeyDiffed] is [SeedKey] (even master key=seed)
+  /// Key may have difference in 2 ways:
+  /// 1) added/removed itself
+  /// 2) added/removed if parent seed added/removed
+  final List<SeedKeyDiffed> deletedKeys;
+  final List<SeedKeyDiffed> addedKeys;
+
+  /// Added and deleted accounts.
+  /// [KeyAccountDiffed] is [KeyAccount] but without any actions.
+  /// Account may have difference in 3 ways:
+  /// 1) added/removed itself
+  /// 2) added/removed if its parent key added/removed
+  /// 3) added/removed if its seed added/removed
+  final List<KeyAccountDiffed> deletedAccounts;
+  final List<KeyAccountDiffed> addedAccounts;
+
+  /// Expand this diff by adding unique elements from [other].
+  SeedListDiffChange expand(SeedListDiffChange other) {
+    return SeedListDiffChange(
+      deletedSeeds:
+          <SeedDiffed>{...other.deletedSeeds, ...deletedSeeds}.toList(),
```

Comment:
Lets add comma before `}.toList` to make it uniform (like `deletedAccounts` and `addedAccounts`).
Now:
```dart
deletedSeeds:
          <SeedDiffed>{...other.deletedSeeds, ...deletedSeeds}.toList(),
```
With comma:
```dart
deletedSeeds:
          <SeedDiffed>{
            ...other.deletedSeeds,
            ...deletedSeeds,
          }.toList(),
```

Reply:
-